### DISCUSSION TITLE
Coupled runs: 033, 034, 035, 036, 037, 038

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#363] [Discussion about coupled runs]
This thread is about the latest coupled runs:
- 033 (#356): include reverting tuning F to retrieve a decent tape recorder
- 034 (#361): same as 033 + `microp_aero_wsub_scale  = 1.0->0.75`
- 035 (#362): same as 033 + `microp_aero_wsub_scale  = 1.0->0.5` + `clubb_gamma_coef = 0.3` +`clubb_gamma_coefb = 0.3`
- 036 (#366): same as 035 +  hard coded **floor value** of wsub is also scaled by 0.5 (`wmixmin = 0.05`)
- 037 (#367): same as 035 + `microp_aero_wsub_scale = 0.5->0.25` + hard coded **floor value** of wsub is set to `wmixmin = 0.025`
- 038 (): same as 036 + `microp_aero_wsub_min            =   0.2->0.`

033 was -1 W/m2 out of balance. We tried to correct that in 034 and 035. 

Looking at the plot:

![032_033_034_035_26g (1)](https://github.com/NCAR/amwg_dev/assets/9723220/b68431b1-c20b-418b-b110-e43e754c75f1)

RESTOM in 035 seems still too cold. I am not sure it is worth to extent it further.
RESTOM in 034 looks a bit better. We could extent it a bit longer. 

Thoughts: @PeterHjortLauritzen @adamrher  ?

**Comment by cecilehannay** [comment]
Sorry, I inverted 34 and 35 in the text but I fixed it now.

**Comment by adamrher** [comment]
I agree with extending 34. Let me look at the clouds in 35 ...

**Reply by adamrher** [reply]
Run 34 SWCF compared to 33 and obs:

![temp_SWCF 34](https://github.com/NCAR/amwg_dev/assets/29961476/922bd75c-786b-4590-9c29-835c5cf527d2)

Unfortunately, I don't like where these clouds are headed. And in particular the further degradation of the DYCOMS region. The thinning of clouds at mid-latitudes is welcome though.

Run 35 SWCF compared to 33 and obs:

![temp_SWCF 35](https://github.com/NCAR/amwg_dev/assets/29961476/b644cfd2-e94f-4f5b-b179-8a5c269af005)

I do like where this is headed. The lowering of the gamma (033->0.30) brings back some DYCOMS clouds while reducing the wscale_factor (1.0->0.5) thins clouds over the storm tracks. Obviously this is not a tenable configurations owing to the -1 W/m2 RESTOM. But I like trading off the cloud forcing, thickening the MarineSc and thinning the storm track clouds -- that improves our biases. I'm wondering if we could further lower the wscale_factor=0.5->0.25 (?) or else maybe we could reduce the fall speed of warm rain?

For reference, the LWCF in 33, 34 and 35, is 24.22 W/m2, 23.87 W/m2 and 23.76, respectively. All of these are acceptable, IMO.

**Comment by cecilehannay** [comment]
Updated timeseries:
![032_033_034_035_26g_newest](https://github.com/NCAR/amwg_dev/assets/9723220/a126bd92-47ac-43b7-b9f5-a709b5b085fc)

**Reply by cecilehannay** [reply]
After reviewing the results of 033, 034 and 035, we started 2 new coupled simulations: 036 and 037

**Reply by cecilehannay** [reply]
Updated timeseries including the 2 new runs:
![032_033_034_035_36_37_26g](https://github.com/NCAR/amwg_dev/assets/9723220/0013f805-b17b-4dee-82a9-6a5ab35fce61)

36 produced bfb results and we stopped that run. I also ran the ADF for 34, 35, and 37. You can find the links in each run issue.

**Comment by PeterHjortLauritzen** [comment]
Tape recorder plots for coupled simulations:

![h2o](https://github.com/NCAR/amwg_dev/assets/8782989/eaa0a163-e5b2-45c7-8574-258be4d739ec)

**Comment by tilmes** [comment]
@cecilehannay  
BURDENDUST to BURDENDUSTdn; 
BURDENPOM to BURDENPOMdn; 
BURDENBC to BURDENBCdn; 
BURDENSEASALT to BURDENSEASALTdn; 
BURDENSO4 to BURDENSO4dn; 
BURDENSOA to BURDENSOAdn;
if available: AODVISstdn (you already have AODVISdn)
